README.md

```diff
@@ -133,7 +133,8 @@ mutation {
         }
       }
       customer:{
-        name: "CUSTOMER_NAME"
+        firstName: "CUSTOMER_FIRSTNAME",
```

Comment:
Are we actually removing the `name` field?  Would be better to support it and mark it as deprecated until our partners have had a chance to change their clients.